Content.Client/Audio/Jukebox/JukeboxMenu.xaml

```diff
@@ -1,18 +1,25 @@
 <ui:FancyWindow xmlns="https://spacestation14.io" xmlns:ui="clr-namespace:Content.Client.UserInterface.Controls"
-                SetSize="400 500" Title="{Loc 'jukebox-menu-title'}">
+xmlns:jb="clr-namespace:Content.Client.Audio.Jukebox"
+SetSize="400 500" Title="{Loc 'jukebox-menu-title'}">
     <BoxContainer Margin="4 0" Orientation="Vertical">
-        <ItemList Name="MusicList" SelectMode="Button" Margin="3 3 3 3"
-        HorizontalExpand="True" VerticalExpand="True" SizeFlagsStretchRatio="8"/>
+        <Label Text="{Loc 'jukebox-menu-songs'}"/>
+        <ScrollContainer HScrollEnabled="False" HorizontalExpand="True" VerticalExpand="True" Margin="13 3 3 3">
```

Comment:
If you remove the `Margin` from this ScrollContainer, then wrap the whole ScrollContainer in a:

```<PanelContainer StyleClasses="PanelBackgroundBaseDark" HorizontalExpand="True" VerticalExpand="True" Margin="13 3 3 3">```

Then, do the same for the other ScrollContainer, you'll get a slightly nicer UI, where the elements don't run together so much:

![image](https://github.com/user-attachments/assets/6223fbec-3eaa-4610-be72-976bcd962268)